Standards/scs-0001-v2-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,309 @@
+`T` is replaced with the slugified title.
+For a document with the number 190, with a major version number 2 and a slugified title `flavor-naming`,
+the resulting file name would be `scs-0190-v2-flavor-naming.md`.
+
+The second digit in `XXXX` describes the track where the document belongs:
+
+| Track  | Number |
+|--------|--------|
+| Global | 0      |
+| IaaS   | 1      |
+| KaaS   | 2      |
+| IAM    | 3      |
+| Ops    | 4      |
+
+In addition to the number, each document has the following metadata,
+embedded in the markdown header.
+
+| Field name      | Requirement                                                                | Description                                                                           |
+|-----------------|----------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| `type`          | REQUIRED                                                                   | one of `Procedural`, `Standard`, or `Decision Record`                                 |
+| `status`        | REQUIRED                                                                   | one of `Proposal`, `Draft`, `Stable`, `Deprecated`, or `Rejected`                     |
+| `track`         | REQUIRED                                                                   | one of `Global`, `IaaS`, `KaaS`, `IAM`, `Ops`                                         |
+| `obsoleted_at`  | REQUIRED if `status` is `Deprecated`                                       | ISO formatted date indicating the date after which the deprecation is in effect       |
+| `stabilized_at` | REQUIRED if `status` was ever `Stable`                                     | ISO formatted date indicating the date after which the document was considered stable |
+| `rejected_at`   | REQUIRED if `status` is `Rejected`                                         | ISO formatted date indicating the date on which the document was rejected             |
+| `replaced_by`   | RECOMMENDED if `status` is `Deprecated` or `Rejected`, FORBIDDEN otherwise | List of documents which replace this document.                                        |
+| `replaces`      | OPTIONAL                                                                   | List of documents which this document replaces.                                       |
+
+### Sections
+
+#### Standard
+
+Each Standard document MUST have the following sections:
+
+- An _Introduction_ providing context on the document and linking to other relevant materials.
+- A _Motivation_ section which details why this document or the thing it describes is necessary.
+- A _Standard_ section containing the actual standardization decision.
+
+In addition, the following OPTIONAL sections should be considered:
+
+- A _Terminology_ section which shortly describes terms used in the document, including possible abbreviations.
```

Comment:
_Terminology_ should be a recommended section, as it increases the quality of a standard because it provides a common vocabulary for all readers.

Reply:
Update in https://github.com/SovereignCloudStack/standards/pull/441/commits/b72701c69882aef9d9ce95d63e2fbb5edc8fd06b